string DataType
string DateTime
int8 MMSI
float64 cog
float64 draft
float64 groundMileAll
float64 groundMileClean
float64 head
float64 headRatio
float64 lat
float64 lon
int8 mmsi
float64 pitch
float64 pitchRatio
float64 roll
float64 rollRatio
float64 seaDepth
float64 sog
float64 sogX
float64 sogY
float64 stw
float64 stwX
float64 stwY
float64 waterMileAll
float64 waterMileClean